src/internal/operations/test/exchange_test.go

```diff
@@ -842,3 +845,333 @@ func testExchangeContinuousBackups(suite *ExchangeBackupIntgSuite, toggles contr
 		})
 	}
 }
+
+type ExchangeRestoreIntgSuite struct {
+	tester.Suite
+	its intgTesterSetup
+}
+
+func TestExchangeRestoreIntgSuite(t *testing.T) {
+	suite.Run(t, &ExchangeRestoreIntgSuite{
+		Suite: tester.NewIntegrationSuite(
+			t,
+			[][]string{tconfig.M365AcctCredEnvs, storeTD.AWSStorageCredEnvs}),
+	})
+}
+
+func (suite *ExchangeRestoreIntgSuite) SetupSuite() {
+	suite.its = newIntegrationTesterSetup(suite.T())
+}
+
+func (suite *ExchangeRestoreIntgSuite) TestRestore_Run_exchangeWithAdvancedOptions() {
+	t := suite.T()
+
+	ctx, flush := tester.NewContext(t)
+	defer flush()
+
+	// a backup is required to run restores
+
+	baseSel := selectors.NewExchangeBackup([]string{suite.its.userID})
+	baseSel.Include(
+		// cannot be run, for the same reason as incremental backups
```

Comment:
nit: put the data type that can't be run so the comment is complete without having to determine what the commented out code does
```suggestion
		// Events cannot be run, for the same reason as incremental backups
```